vite.config.js

```diff
@@ -1,16 +1,21 @@
 import { defineConfig } from 'vite'
-import path from 'path'
-import { config } from './vite.config.server'
+import { dirname, resolve } from 'path'
+import { fileURLToPath } from 'url'
+import react from '@vitejs/plugin-react'
+import mdx from '@mdx-js/rollup'
+import rehypeSlug from 'rehype-slug'
+
+const dir = dirname(fileURLToPath(import.meta.url))
 
 export default defineConfig({
-  plugins: config.plugins,
+  plugins: [react(), mdx({ rehypePlugins: [rehypeSlug] })],
```

Comment:
We no longer need the extra `vite.config.server.js` due to the changes in `prerender.js`, so its config has moved in here.